tests/tests_data/streaming/test_dataloader.py

```diff
@@ -29,9 +29,17 @@ def state_dict(self, *args, **kwargs):
     def load_state_dict(self, state_dict):
         self.counter = state_dict["counter"]
 
+    def set_epoch(self, current_epoch):
+        pass
+
+
+class TestCombinedStreamingDataset(CombinedStreamingDataset):
+    def _check_datasets(self, datasets) -> None:
+        pass
```

Comment:
Mocking the method in a one-liner would be cleaner than subclassing IMO

Reply:
I kinda disagree. You can subclass once or patch in every single class. The first option results in less code.